tests/TestCase/Command/BakeFixtureFactoryCommandTest.php

```diff
@@ -63,6 +65,18 @@ public function testGetFileName()
         $this->assertSame('ModelFactory.php', $this->FactoryCommand->getFactoryFileName($name));
     }
 
+    public function testGetPath()
+    {
+        $args = new Arguments([], [], []);
```

Comment:
This variable is required to pass to `getPath()` even though the function doesn't use that parameter, so this is why I'm wondering if it can be removed? Thanks!

Reply:
See the comment above, $args should remain